src/Microsoft.AspNetCore.Mvc.Formatters.Xml/XmlSerializerOutputFormatter.cs

```diff
@@ -197,6 +217,17 @@ protected virtual XmlSerializer CreateSerializer(Type type)
             }
         }
 
+        /// <summary>
+        /// Serializes value using the passed in <see cref="XmlSerializer"/> and <see cref="XmlWriter"/>.
+        /// </summary>
+        /// <param name="xmlSerializer"></param>
+        /// <param name="xmlWriter"></param>
+        /// <param name="value"></param>
+        protected virtual void Serialize(XmlSerializer xmlSerializer, XmlWriter xmlWriter, object value)
```

Comment:
Why does this class need this `virtual` method but `XmlDataContractSerializerOutputFormatter` doesn't need the equivalent?

Reply:
I wasn't sure about the scenario for which they will use the Serialize in this case (for example, changing namespaces as in XmlSerializer is not possible).

Reply:
:ok: